locksmith/src/controllers/v2/priceController.ts

```diff
@@ -16,11 +16,12 @@ export const amount: RequestHandler = async (request, response) => {
     ? erc20Address
     : undefined
 
-  const result = await pricingOperations.getDefiLammaPrice({
-    network,
-    amount,
-    address,
-  })
+  const result =
+    await pricingOperations.getUsdLockPricingFromSettingOrConverted({
+      network,
+      amount,
+      address,
+    })
```

Comment:
I'm confused by the name here. Let's keep the more clearer function if we are just calling that conditionally underneath?

Reply:
renamed to `getUsdLockPricing`

Reply:
No, I mean let's call the defilammaPrice since that's what this is calling underneath since this has nothing to do with lock.